Getting a bunch of undefined method calls in Moodle's PHPUnit tests

Hi, with the latest release, 8.0.0, we are seeing these PHPUnit test errors on Moodle 5.1.4:

```
There were 7 errors:

1) block_massaction\massaction_test::test_mass_delete_modules
Error: Call to undefined method core_courseformat\local\cmactions::delete()

/home/runner/work/moodle/moodle/public/blocks/massaction/classes/actions.php:541
/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:139

2) block_massaction\massaction_test::test_mass_move_modules_to_new_section
Error: Call to undefined method core_courseformat\local\cmactions::move_end_section()

/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:894
/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:224

3) block_massaction\massaction_test::test_mass_hide_unhide_modules
Error: Call to undefined method core_courseformat\local\sectionactions::set_visibility()

/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:394

4) block_massaction\massaction_test::test_mass_duplicate_modules
Error: Call to undefined method core_courseformat\local\cmactions::move_end_section()

/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:894
/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:450

5) block_massaction\massaction_test::test_mass_duplicate_modules_failing_in_the_middle
Error: Call to undefined method core_courseformat\local\cmactions::duplicate()

/home/runner/work/moodle/moodle/public/blocks/massaction/classes/actions.php:178
/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:543

6) block_massaction\massaction_test::test_mass_duplicate_modules_to_course
Error: Call to undefined method core_courseformat\local\cmactions::move_end_section()

/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:894
/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:611

7) block_massaction\massaction_test::test_duplicate_to_course_with_filter_sections_different_course_hook
Error: Call to undefined method core_courseformat\local\cmactions::move_end_section()

/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:894
/home/runner/work/moodle/moodle/public/blocks/massaction/tests/massaction_test.php:946

ERRORS!
Tests: 24492, Assertions: 159878, Errors: 7, PHPUnit Deprecations: 6881, Notices: 48, Skipped: 456, Incomplete: 5.
Error: Process completed with exit code 2.
```

--- a/version.php
+++ b/version.php
@@ -24,9 +24,9 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version = 2026042700;
-$plugin->requires  = 2024042000;
+$plugin->version = 2026052200;
+$plugin->requires  = 2026042000;
 $plugin->component = 'block_massaction';
 $plugin->maturity = MATURITY_STABLE;
-$plugin->release = 'v8.0.0';
+$plugin->release = 'v8.0.1';
 $plugin->supported = [502, 502];

